common/djangoapps/static_replace/test/test_static_replace.py

```diff
@@ -9,7 +9,8 @@
 import ddt
 import pytest
 from django.test import override_settings
-from django.utils.http import urlencode, urlquote
+from urllib.parse import quote
+from django.utils.http import urlencode
```

Comment:
Suggestion: we can instead use `urllib.parse.urlencode` as well.

Reply:
Also, is isort happy with the import order here?  Not a big problem, but might as well fix it now rather than try to figure it out later.